strapiExperiment/app/.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
will this conflict with eslint? Eslint has oppinions on indent size. That actually might be prettier.

Reply:
I don't know, this was auto-generated by strapi.  I imagine we can tear it out when we implement eslint if they do conflict.

Reply:
gotchya,... maybe leave it as is since its boiler plate ?